instaclustr/resource_cluster.go

```diff
@@ -1176,7 +1176,12 @@ func resourceClusterRead(d *schema.ResourceData, meta interface{}) error {
 		}
 		d.Set("data_centre", cluster.DataCentres[0].Name)
 		d.Set("cluster_network", cluster.DataCentres[0].CdcNetwork)
+
+		if err := deleteAttributesConflict(resourceCluster().Schema, d, "data_centre"); err != nil {
```

Comment:
We need to delete attributes conflicting with `data_centre` if for whatever reason the actual infrastructure is single-DC and the terraform state is multi-DC